src/bit_vec.rs

```diff
@@ -42,6 +43,19 @@ impl<O: BitOrder, T: BitStore + ToByteSlice> Encode for BitSlice<O, T> {
 	}
 }
 
+fn reverse_endian(vec_u8: &mut Vec<u8>, size_of_t: usize) {
```

Comment:
Some docs would be nice.

Reply:
oh yes, I added some, does it need more ?

Reply:
Ty